experiments/classification/cifar10/wideresnet.py

```diff
@@ -8,11 +8,14 @@
 from torch_uncertainty.optimization_procedures import get_procedure
 
 if __name__ == "__main__":
-    root = Path(__file__).parent.absolute().parents[2]
-
     args = init_args(WideResNet, CIFAR10DataModule)
+    if args.root == "./data/":
+        root = Path(__file__).parent.absolute().parents[2]
+    else:
+        root = Path(args.root)
```

Comment:
I think this behavior might appear strange to users. I would suggest replacing ``"./data/"`` with ``"default"`` or even ``None`` if this is to be the default behavior.

Reply:
I fully agree; I just normalized what had been done in dev.